.github/workflows/config/spelling_allowlist.txt

```diff
@@ -117,8 +120,10 @@ comparator
 comparators
 composability
 composable
+const
```

Comment:
Is this coming from code? Should it be backquoted? Just wondering...

Reply:
this is coming from code snippets in the docs markdown. I don't know how else to get it through the spell check

Reply:
I suspect you just need to check that your `code-block` sections are indented with at least 4 spaces.

Also, just a tip in case you didn't know: you can run `bash scripts/run_all_spelling.sh -d` locally to check the changes. The `-d` means to just check the files that are different than on the `main` branch.